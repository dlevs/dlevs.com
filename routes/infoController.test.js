'use strict';

const { getControllerRenderData } = require('../tests/testLib/testUtils');
const infoController = require('./infoController')();

describe('index', () => {
	test('data passed to template', async () => {
		const data = await getControllerRenderData(infoController.index);
		expect(data.ctx.body).toBeDefined();

		delete data.ctx.body.date;
		delete data.ctx.body.serverStartDate;
		delete data.ctx.body.lastCommit;

		expect(data.ctx.body).toMatchSnapshot();
	});
});
