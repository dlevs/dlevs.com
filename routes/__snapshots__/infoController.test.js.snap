// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index data passed to template with ctx {} 1`] = `
Object {
  "ctx": Object {
    "body": Object {
      "appVersion": "1.0.0",
      "environment": "test",
      "nodeVersion": "v8.11.2",
    },
    "query": Object {},
    "render": [Function],
    "state": Object {
      "CANONICAL_URL": "DUMMY_CANONICAL_URL",
    },
  },
  "renderedData": undefined,
}
`;
